CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### :rotating_light: Breaking changes
 - ...
 
+## [v0.20.0] - 2024-07-07
+
+This version is same as v0.19.0.
```

Comment:
```suggestion
This version is created by mistake and is the same as v0.19.0.
We left both versions as-is so that it won't break existing clients who already use v0.20.
```

Reply:
I copied it to https://github.com/reviewdog/reviewdog/releases/tag/v0.20.0.